ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -8,8 +8,197 @@ periodics:
+        name: osde2e-credentials
+        readOnly: true
+      - mountPath: /usr/local/osde2e-stage-ccs
+        name: osde2e-stage-ccs
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: osde2e-common
+      secret:
+        secretName: osde2e-common
+    - name: osde2e-credentials
+      secret:
+        secretName: osde2e-credentials
+    - name: osde2e-stage-ccs
+      secret:
+        secretName: osde2e-stage-ccs
+- agent: kubernetes
+  cluster: build03
+  cron: 0 */12 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
+  name: osde2e-aws-stage-e2e-machine-type-enablement-c5d
+  spec:
+    containers:
+    - args:
+      - test
+      - --secret-locations
+      - $(SECRET_LOCATIONS)
+      - --configs
+      - $(CONFIGS)
+      command:
+      - /osde2e
+      env:
+      - name: CCS_OVERWRITE
+        value: "true"
+      - name: CHANNEL
+        value: stable
+      - name: CONFIGS
+        value: aws,stage,e2e-suite,region-random
+      - name: OCM_CCS
+        value: "true"
+      - name: OCM_COMPUTE_MACHINE_TYPE
+        value: random
+      - name: OCM_COMPUTE_MACHINE_TYPE_REGEX
+        value: c5d
+      - name: SECRET_LOCATIONS
+        value: /usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-stage-ccs
+      image: quay.io/app-sre/osde2e
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/osde2e-common
+        name: osde2e-common
+        readOnly: true
+      - mountPath: /usr/local/osde2e-credentials
+        name: osde2e-credentials
+        readOnly: true
+      - mountPath: /usr/local/osde2e-stage-ccs
+        name: osde2e-stage-ccs
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: osde2e-common
+      secret:
+        secretName: osde2e-common
+    - name: osde2e-credentials
+      secret:
+        secretName: osde2e-credentials
+    - name: osde2e-stage-ccs
+      secret:
+        secretName: osde2e-stage-ccs
+- agent: kubernetes
+  cluster: build03
+  cron: 0 */12 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
+  name: osde2e-aws-stage-e2e-machine-type-enablement-m5d
+  spec:
+    containers:
+    - args:
+      - test
+      - --secret-locations
+      - $(SECRET_LOCATIONS)
+      - --configs
+      - $(CONFIGS)
+      command:
+      - /osde2e
+      env:
+      - name: CCS_OVERWRITE
+        value: "true"
+      - name: CHANNEL
+        value: stable
+      - name: CONFIGS
+        value: aws,stage,e2e-suite,region-random
+      - name: OCM_CCS
+        value: "true"
+      - name: OCM_COMPUTE_MACHINE_TYPE
+        value: random
+      - name: OCM_COMPUTE_MACHINE_TYPE_REGEX
+        value: m5d.
```

Comment:
typo?
```suggestion
        value: m5d
```

Reply:
disregard...i was assuming `m5d` as the string and didn't realize it would be something like `m5d.xlarge`